posthog/hogql/test/test_expr_parser.py

```diff
@@ -0,0 +1,96 @@
+from posthog.hogql.expr_parser import ExprParserContext, translate_hql
+from posthog.test.base import APIBaseTest, ClickhouseTestMixin  # , test_with_materialized_columns
+
+
+class TestExprParser(ClickhouseTestMixin, APIBaseTest):
+    def test_hogql_literals(self):
+        self.assertEqual(translate_hql("1 + 2"), "plus(1, 2)")
+        self.assertEqual(translate_hql("-1 + 2"), "plus(-1, 2)")
+        self.assertEqual(translate_hql("-1 - 2 / (3 + 4)"), "minus(-1, divide(2, plus(3, 4)))")
+        self.assertEqual(translate_hql("1.0 * 2.66"), "multiply(1.0, 2.66)")
+        self.assertEqual(translate_hql("'string'"), "'string'")
+        self.assertEqual(translate_hql('"string"'), "'string'")
+
+    # @test_with_materialized_columns(["$browser"])
```

Comment:
Why's this commented out?

Reply:
Cause it was broken :). The big cause was that the test is now being run twice, once with and once without materialized columns. I changed it to call `materialize` directly before the test, and it seems to work now.